src/App/authActions.ts

```diff
@@ -12,6 +14,23 @@ export const authError = (e: Error): unknown => ({
   error: e,
 });
 
+function warningNotif(id: string, message: string) {
+  store.addNotification({
+    title: id,
+    message: message,
```

Comment:
you don't need `message:message`, you can just use `message` here, json works that way